android/app/src/main/java/org/fossasia/openevent/utils/NotificationUtil.java

```diff
@@ -37,12 +37,15 @@ public static Completable createNotification(Session session, Context context) {
             } else {
                 zonedDateTime.minusHours(10);
             }
-            Intent myIntent = new Intent(context, NotificationAlarmReceiver.class);
-            myIntent.putExtra(ConstantStrings.SESSION, session.getId());
-            PendingIntent pendingIntent = PendingIntent.getBroadcast(context, 0, myIntent, PendingIntent.FLAG_UPDATE_CURRENT);
-
-            AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
-            alarmManager.set(AlarmManager.RTC, zonedDateTime.toInstant().toEpochMilli(), pendingIntent);
+            // Checking if the event time is before the current time
+            if (zonedDateTime.isBefore(ZonedDateTime.now())) {
```

Comment:
It should be the opposite logic. If the event time is after current time, the notification should be shown